docs/markdown/Builtin-options.md

```diff
@@ -384,10 +384,10 @@ command line: `-Dfoo:default_library=static`.
 The value is overridden in this order:
 - `opt=value` from parent project's `default_options`
 - `opt=value` from subproject's `default_options`
-- `subp:opt=value` from parent project's default options
-- `opt=value` from `subproject()` `default_options`
 - `opt=value` from machine file
 - `opt=value` from command line
+- `subp:opt=value` from parent project's default options
+- `opt=value` from `subproject()` `default_options`
```

Comment:
This seems incorrect. The command line should override anything set in project files. (given the description this list is sorted on increasing precedence)

Reply:
The problem here seems to be that the table collates two different things making it hard to comprehend. Maybe it should be split in two, with a structure like:

- Per-subproject settings always override the default value, no matter how or where they are defined
- When configuring for the first time, values for options (and overrides) are used in the following order

And maybe have the array in decreasing order of priority, as that seems to be the more logical order.